packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -67,9 +67,9 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
 
   /**
    * The default value of the text in the SearchBox, in the case of an uncontrolled component.
-   * Up till now, this has not been implemented, deprecating. Will re-implement if uncontrolled
+   * Deprecating prop since so far, this has not been implemented. Will un-deprecate if uncontrolled
```

Comment:
wording

Reply:
I actually think this is an improvement over the previous wording--what would you suggest?

Reply:
Updated again